raiden-dapp/package.json

```diff
@@ -56,7 +56,7 @@
     "eslint-plugin-jsdoc": "^15.9.5",
     "eslint-plugin-prettier": "^3.1.1",
     "eslint-plugin-vue": "^5.2.3",
-    "eslint-plugin-vue-i18n": "^0.2.3",
+    "eslint-plugin-vue-i18n": "^0.3.0",
```

Comment:
Updated manually in favor of https://github.com/kazupon/eslint-plugin-vue-i18n/issues/25 so we can ignore `v-icon` tags for the `vue-i18n/no-raw-text` rule